src/vfc_ddebug

```diff
@@ -1,9 +1,7 @@
-#!/bin/sh
-''''exec setarch $(uname -m) -R python3 -u "$0" "$@" #'''
-# This hack is an ugly but portable alternative to #!/usr/bin/env -S python3 -u
-# we call setarch -R to disable ASLR, so that addresses during reference run
```

Comment:
can you move this comment to the function disable_ASLR to explain why this is needed during delta_debug

Reply:
Done.

Reply:
Done.